components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -285,18 +290,7 @@ - (BOOL)isMobileLandscape {
 }
 
 - (BOOL)shouldPresentFullScreen {
-  return [self isAccessibilityMode] || [self isMobileLandscape];
-}
-
-/**
- The drawer height factor defines how much percentage of the screen space the drawer will take up
- when displayed. The expected range is 0 - 1 (0% - 100%).
-
- Default value is 0.5. If VoiceOver is enabled, or the mobile device is in landscape,
- the default value becomes 1.0.
- */
-- (CGFloat)initialDrawerFactor {
-  return [self shouldPresentFullScreen] ? 1 : (CGFloat)0.5;
+  return [self isAccessibilityMode] || [self isMobileLandscape] || _initialDrawerFactor >= 1;
```

Comment:
i think there is probably a better comparison than _initialDrawerFactor >=1. Reason being is that if the initial drawer factor is close to 1, i think the drawer still goes to full screen because it adjusts to full screen if its in the middle of transition the header to cover the safe area and the status bar.

please open a follow up bug to research this is a follow up PR.